### ISSUE TITLE
Lad være med at have hemmelige kommandoer på den hemmelige kanal

### ISSUE THREAD

**Issue by nqpz** [Athas/EggsML#136]
Eksempel: Vi har både en `jeg`-kommando i `#diku` og `#eggsml`, og den i `#eggsml` kan lidt flere ting, også ting der ikke har noget med frokost at gøre.

Jeg foreslår en SAMLING AF KOMMANDOERNE for så vidt muligt.

**Comment by nqpz** [comment]
Rettelse: Der er ikke en `jeg`-kommando på begge kanaler. Men der er en `hvad`-kommando. Og nogle andre ting. Jeg vil i hvert fald gerne have ryddet op i det.

**Comment by sshine** [comment]
Make `hvad` great again!

**Comment by sshine** [comment]
Er der overhovedet brug for kanalspecifikke kommandoer?

**Comment by svip** [comment]
Jaaa!  Jeg vil kunne tilmelde mig eggsml via #diku-kanalen!  Der er dog stadig kanalspecifikke kroge, da concieggs siger en masse lort der.

**Comment by nqpz** [comment]
Ja, det er mest bare gamle frokostspecifikke kommandoer der ikke bliver brugt længere.

De må godt hygge sig på deres egen kanal.

**Comment by sshine** [comment]
> Jeg vil kunne tilmelde mig eggsml via #diku-kanalen!

Det ser jeg ikke et problem med, eftersom ingen tilmelder sig frokostordningen længere.

> Der er dog stadig kanalspecifikke kroge

Er der overhovedet brug for kanalspecifikke kroge?
